pkg/cmd/pulumi/stack_ls.go

```diff
@@ -34,6 +34,7 @@ import (
 	"github.com/pulumi/pulumi/pkg/v3/backend/state"
 	cmdBackend "github.com/pulumi/pulumi/pkg/v3/cmd/pulumi/backend"
 	"github.com/pulumi/pulumi/pkg/v3/cmd/pulumi/cmd"
+	cmdTime "github.com/pulumi/pulumi/pkg/v3/cmd/pulumi/time"
```

Comment:
is time the best package name for this if we're gonna have to "cmdTime" it forever?

Reply:
Fair point. `format` isn't great (as in `format.Time`), `cmdtime` is a bit ugly case-wise but maybe the best compromise? Open to all suggestions.

Reply:
I've gone with `cmd/time.go` so now it's `cmd.FormatTime`. I think this makes sense -- thoughts?